styx-cli/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java

```diff
@@ -20,19 +20,30 @@
 
 package com.spotify.styx.client;
 
+import static com.google.common.collect.Iterables.getLast;
+import static java.util.Arrays.asList;
+import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.startsWith;
 import static org.junit.Assert.assertThat;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.timeout;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+import com.google.common.collect.ImmutableList;
 import com.spotify.apollo.Client;
 import com.spotify.apollo.Request;
 import com.spotify.apollo.Response;
+import com.spotify.apollo.Status;
+import com.spotify.styx.api.Api;
+import com.spotify.styx.model.Schedule;
+import com.spotify.styx.model.Workflow;
+import com.spotify.styx.model.WorkflowConfiguration;
+import com.spotify.styx.serialization.Json;
 import java.util.concurrent.CompletableFuture;
 import junitparams.JUnitParamsRunner;
 import junitparams.Parameters;
+import okhttp3.HttpUrl;
```

Comment:
com.squareup.okhttp.HttpUrl

Reply:
Not sure what the difference is...

Reply:
We have both okhttp and okhttp3 as dependency. :O